thirdeye/thirdeye-pinot/src/main/resources/migrations.xml

```diff
@@ -553,4 +552,27 @@
     </createTable>
   </changeSet>
 
+   <changeSet id="21" author="npawar">
+  <!-- WebappConfigs -->
+    <createTable tableName="webapp_configs">
+      <column name="id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="config_id" type="bigint">
```

Comment:
can this config_id be present in more than one rows in this table, if not then remove it and use 'id'.